packages/fxa-payments-server/src/components/fields/index.tsx

```diff
@@ -214,19 +214,25 @@ export const Input = withLocalization(UnwrappedInput);
 
 type StripeElementProps = { onValidate?: OnValidateFunction } & FieldProps & {
     component: any;
+    getString: Function | undefined;
   } & ReactStripeElements.ElementProps;
 
 export const defaultStripeElementValidator: OnValidateFunction = (
   value,
   focused,
-  props
+  props,
+  getString
 ) => {
   if (!value || value.empty) {
     if (props.required) {
       return {
         value,
         valid: false,
-        error: focused ? null : `${props.label} is required`,
+        error: focused
+          ? null
+          : `${props.label} ${
```

Comment:
We can't handle translated strings with concatenation like this. The "is required" string must have a placeholder like "${fieldLabel} is required` and allow the localizer to swap word order as necessary.

Reply:
Gotcha, I'll update